File: unit-3/sprint-3/day-2/assignments/Perform_background_JobIntentService/app/src/main/java/com/example/perform_background_jobintentservice/MainActivity.java
package com.example.perform_background_jobintentservice;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {
    private Button mBtnJob1, mBtnJob2;
    private Intent intent;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        initView();
    }

    private void initView() {
        mBtnJob1 = findViewById(R.id.btnJobId1);
        mBtnJob2 = findViewById(R.id.btnJobId2);
        intent = new Intent(MainActivity.this,MyIntentService.class);

        mBtnJob1.setOnClickListener(v -> {
            intent.putExtra("JobId","1");
            MyIntentService.enqueueWork(this,intent);
        });

        mBtnJob2.setOnClickListener(v -> {
            intent.putExtra("JobId","2");
            MyIntentService.enqueueWork(this,intent);
        });
    }


}

File: unit-3/sprint-4/day-3/we/ScopingFunction/app/src/main/java/com/example/scopingfunction/MainActivity.kt
package com.example.scopingfunction

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        launchFragment()
    }

    private fun launchFragment() {
        val fragmentManager = supportFragmentManager
        val fragmentTransaction = fragmentManager.beginTransaction()
        fragmentTransaction.add(R.id.flContainer,UserDetailsFragment(),"userdetail").commit()
    }
}